sdk/identity/identity/src/tokenCache/requireMsalNodeExtensions.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// TODO:
+// Either drop Node 8 or re-enable it.
+
+/**
+ * @internal
+ */
+export const isNode8 = process.versions.node[0] === "8";
```

Comment:
Feel free to ignore if this code only runs on `node` via replacements but `process` will not defined in the browser

Reply:
This code only runs on Node 👍